HedgeModManager/Languages/ja-JP.xaml

```diff
@@ -0,0 +1,241 @@
+    <system:String x:Key="ModDescriptionUIMadeBy">Made by</system:String>
+    <system:String x:Key="ModDescriptionUIMadeOn">Made on</system:String>
+    <system:String x:Key="ModDescriptionUIDate">on</system:String>
+
+    <!-- Mod Downloader Strings -->
+    <system:String x:Key="ModDownloaderFailed">MODダウンロードに失敗</system:String>
+    <system:String xml:space="preserve" x:Key="ModDownloaderWebError">ダウンロード中にエラーが発生しました。サーバーがダウンしているか、過負荷になっている可能性があります。後でもう一度お試しください。 &#x0a;&#x0a;詳細: {0}</system:String>
+    <system:String x:Key="ModDownloaderNoGame">ゲームの検出に失敗</system:String>
+    <system:String x:Key="ModDownloaderNoGameMes">Hedge Mod Managerは {0} のインストール先を見つけられませんでした。</system:String>
+
+    <!-- Mod Updates Window Strings -->
+    <system:String x:Key="ModUpdatesChangelog"    >更新履歴</system:String>
+    <system:String x:Key="ModDownloadsDescription">詳細</system:String>
+    <system:String x:Key="ModUpdatesTitle"        >MODアップデート</system:String>
+    <system:String x:Key="ModDownloadsTitle"      >MODダウンロード</system:String>
+
+    <!-- Exception Window Strings -->
+    <system:String x:Key="ExceptionWindowTitle"           >Hedge Mod Managerのクラッシュ</system:String>
+    <system:String x:Key="ExceptionWindowCrash"           >Hedge Mod Managerが例外をスローしました。</system:String>
+    <system:String x:Key="ExceptionWindowUnhandled"       >Hedge Mod Managerにエラーが発生しました。</system:String>
+    <system:String x:Key="ExceptionWindowButtonCopyLog"   >ログをクリップボードにコピー</system:String>
+    <system:String x:Key="ExceptionWindowButtonIgnore"    >無視して閉じる</system:String>
+    <system:String x:Key="ExceptionWindowButtonReport"    >Githubでレポートを提出する</system:String>
+    <system:String x:Key="ExceptionWindowNewUpdateTitle"  >アップデートが利用可能です!</system:String>
+    <system:String x:Key="ExceptionWindowNewUpdateText" xml:space="preserve" >このHedge Mod Managerのバージョンは古いようです。&#x0a;まずはアップデートを行い、問題が発生するかどうかを確認することを強くお勧めします。&#x0a;今すぐアップデートしますか？</system:String>
+    <system:String x:Key="ExceptionWindowUpdateErrorTitle">アップデートエラー!</system:String>
+    <system:String x:Key="ExceptionWindowUpdateErrorText" xml:space="preserve" >Hedge Mod Manager のアップデートができませんでした!&#x0a;最新版を手動でダウンロードしてください。</system:String>
+
+    <!-- Themes -->
+    <system:String x:Key="ThemeDark"  >ダーク</system:String>
+    <system:String x:Key="ThemeDarker">漆黒</system:String>
+    <system:String x:Key="ThemeOLED"  >OLED</system:String>
+    <system:String x:Key="ThemeLight" >ライト</system:String>
+
+    <!-- Games -->
+    <system:String x:Key="GameSonicGenerations">Sonic Generations - ソニックジェネレーションズ</system:String>
+    <system:String x:Key="GameSonicLostWorld">Sonic Lost World - ソニックロストワールド</system:String>
+    <system:String x:Key="GameSonicForces">Sonic Forces - ソニックフォース</system:String>
+    <system:String x:Key="GamePuyoPuyoTetris2">Puyo Puyo Tetris 2 - ぷよぷよテトリス2</system:String>
+    <system:String x:Key="GameTokyo2020">Olympic Games Tokyo 2020</system:String>
```

Comment:
Tokyo 2020 has a Japanese name, referring to it's [SteamDB entry](https://steamdb.info/app/981890/info/).
```suggestion
    <system:String x:Key="GameTokyo2020">Olympic Games Tokyo 2020 - 東京2020オリンピック</system:String>
```